libraryBuilds/com.fasterxml.jackson.core/com.fasterxml.jackson.core-jackson-databind/src/test/java/com/fasterxml/jackson/custom/GenericsTest.java

```diff
@@ -0,0 +1,193 @@
+package com.fasterxml.jackson.custom;
```

Comment:
1. Did you author this file?  Or did you get it from somewhere?
2. Unless this is Jackson's own test, we should not put in the Jackson package.  If it is Jackson's own unit test, we should not copy it into our repo and instead download and reference the unit tests in Jackson instead.  See https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/pull/45